.github/workflows/build.yml

```diff
@@ -48,10 +49,10 @@ jobs:
           file: Dockerfile.slim
           push: true
           tags: |
-            "${{ env.CONTAINER_REGISTRY }}/${{ steps.extract_repo_name.outputs.REPO }}:${{ github.ref_name }}-latest-unvalidated"
-            "${{ env.CONTAINER_REGISTRY }}/${{ steps.extract_repo_name.outputs.REPO }}:${{ github.ref_name }}-${{ github.sha }}-${{ steps.get_date.outputs.DATETIME }}-unvalidated"
-            "${{ env.CONTAINER_REGISTRY }}/${{ steps.extract_repo_name.outputs.REPO }}:${{ github.ref_name }}-latest-unvalidated-slim"
-            "${{ env.CONTAINER_REGISTRY }}/${{ steps.extract_repo_name.outputs.REPO }}:${{ github.ref_name }}-${{ github.sha }}-${{ steps.get_date.outputs.DATETIME }}-unvalidated-slim"
+            "${{ env.CONTAINER_REGISTRY }}/${{ steps.build_variables.outputs.REPO }}:${{ github.ref_name }}-latest-unvalidated"
```

Comment:
Potentially stupid question - why is the slim Dockerfile tagged as slim and normal? Is the other Dockerfile not the "unslim" version?

Reply:
Also unrelated to this PR, I noticed PR builds are building containers - is that intentional?

Reply:
Cheers Matt.

> why is the slim Dockerfile tagged as slim and normal?
This is the way it has always been and I guess it makes it explicit.
The same container image SHA will have both "none" and slim tags - so one can see that there aren't 3 versions (none, slim, ubuntu)...?

See: https://console.cloud.google.com/artifacts/docker/spinnaker-community/us/docker/orca

> PR builds are building containers - is that intentional?
Yes, this exercises Dockerfile related changes which is helpful for services doing a bit of extra work like downloading binaries, running scripts etc. The images aren't pushed.
It would be ideal to enable docker layer caching in the action and we probably can now that images are seeded in GAR. 
Maybe someone else can pick this up though as it's not blocking a 1.27.0 release?

Reply:
Cool, I just know there used to be Dockerfile and Dockerfile.slim, wasn't sure if there was an important distinction there.

And sounds good, just checking we're not burning resources/money accidentally ;)